src/screens/OtherProfile/index.js

```diff
@@ -591,7 +683,7 @@ const OtherProfile = () => {
           onSelect={onBlocking}
           refBlockUser={blockUserRef}
           username={username}
-          isBlocker={isBlocking}
+          isBlocker={blockStatus.blocker}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The block status of a user is now determined by `blockStatus.blocker` instead of `isBlocking`. Make sure that this new variable correctly reflects the block status.
```diff
+ isBlocker={blockStatus.blocker}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->